src/js/common/schemaform/definitions/ssn.js

```diff
@@ -1,9 +1,13 @@
 import { validateSSN } from '../validation';
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+import SSNWidget from '../widgets/SSNWidget';
+import { TextWidget } from '../widgets';
 
 export const schema = commonDefinitions.ssn;
 
 export const uiSchema = {
+  'ui:widget': SSNWidget,
+  'ui:reviewWidget': TextWidget,
```

Comment:
Is `TextWidget` right? I'm getting the following on review:

---
![image](https://cloud.githubusercontent.com/assets/12970166/23002372/40205182-f39f-11e6-89f9-87465cfaa769.png)

---
The SSN is editable prematurely.

Also, I did enter dashes that got stripped away. I'm sure that was the intention, but what if we could display it here like `XXX-XX-XXXX` even if they didn't enter it that way, then send it to the back end numbers-only? That's probably more a question for @goldenmeanie, eh?